src/Microsoft.PowerShell.Commands.Utility/commands/utility/MatchString.cs

```diff
@@ -1026,6 +1067,13 @@ public string[] LiteralPath
         [Parameter]
         public SwitchParameter SimpleMatch { get; set; }
 
+        /// <summary>
+        /// Gets or sets a value indicating if an undecorated string should be outputted.
```

Comment:
```suggestion
        /// Gets or sets a value indicating whether Select-String should emit strings with matches as plain strings rather than MatchInfo objects.
```